GitHub CI fails on PostgreSQL link errors in VS 2019 x86-portable

https://github.com/minetest/minetest/runs/782603002

```
database-postgresql.obj : error LNK2001: unresolved external symbol _PQprepare [D:\a\minetest\minetest\src\minetest.vcxproj]
database-postgresql.obj : error LNK2001: unresolved external symbol _PQgetvalue [D:\a\minetest\minetest\src\minetest.vcxproj]
database-postgresql.obj : error LNK2001: unresolved external symbol _PQresultStatus [D:\a\minetest\minetest\src\minetest.vcxproj]
database-postgresql.obj : error LNK2001: unresolved external symbol _PQreset [D:\a\minetest\minetest\src\minetest.vcxproj]
database-postgresql.obj : error LNK2001: unresolved external symbol _PQstatus [D:\a\minetest\minetest\src\minetest.vcxproj]
database-postgresql.obj : error LNK2001: unresolved external symbol _PQserverVersion [D:\a\minetest\minetest\src\minetest.vcxproj]
database-postgresql.obj : error LNK2001: unresolved external symbol _PQerrorMessage [D:\a\minetest\minetest\src\minetest.vcxproj]
database-postgresql.obj : error LNK2001: unresolved external symbol _PQexecPrepared [D:\a\minetest\minetest\src\minetest.vcxproj]
database-postgresql.obj : error LNK2001: unresolved external symbol _PQfinish [D:\a\minetest\minetest\src\minetest.vcxproj]
database-postgresql.obj : error LNK2001: unresolved external symbol _PQresultErrorMessage [D:\a\minetest\minetest\src\minetest.vcxproj]
database-postgresql.obj : error LNK2001: unresolved external symbol _PQexec [D:\a\minetest\minetest\src\minetest.vcxproj]
database-postgresql.obj : error LNK2001: unresolved external symbol _PQgetlength [D:\a\minetest\minetest\src\minetest.vcxproj]
database-postgresql.obj : error LNK2001: unresolved external symbol _PQclear [D:\a\minetest\minetest\src\minetest.vcxproj]
database-postgresql.obj : error LNK2001: unresolved external symbol _PQconnectdb [D:\a\minetest\minetest\src\minetest.vcxproj]
database-postgresql.obj : error LNK2001: unresolved external symbol _PQntuples [D:\a\minetest\minetest\src\minetest.vcxproj]
database-postgresql.obj : error LNK2001: unresolved external symbol _PQping [D:\a\minetest\minetest\src\minetest.vcxproj]
D:\a\minetest\minetest\bin\Release\minetest.exe : fatal error LNK1120: 16 unresolved externals [D:\a\minetest\minetest\src\minetest.vcxproj]
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -261,6 +261,7 @@ jobs:
           cmake ${{matrix.config.generator}}  `
           -DCMAKE_TOOLCHAIN_FILE="${{ github.workspace }}\vcpkg\scripts\buildsystems\vcpkg.cmake"  `
           -DCMAKE_BUILD_TYPE=Release  `
+          -DENABLE_POSTGRESQL=OFF  `
           -DRUN_IN_PLACE=${{ contains(matrix.type, 'portable') }} .
 
       - name: Build

